plugins/inputs/ssl/README.md

```diff
@@ -0,0 +1,35 @@
+# Example Input Plugin
+
+This input plugin will return how many days left for a SSL cert to expire.
+
+### Configuration:
+
+```
+# SSL request given a server, a Port, a timeout and a skipverify flag
+[[inputs.check_ssl]]
+  ## Server (default github.com)
+  servers = ["github.com"]
+  ## Set response_timeout (default 5 seconds)
+  response_timeout = 5
+  ## Port (Default 443)
+  port = "443"
+  ## SSL Skip Verification of certificates
+  skip_verify = false
```

Comment:
This should probably not be an option at all, and should always behave as `skip_verify = true`.
We're not sending any data over the connection, thus we don't need it to be secure. The purpose of the check it to check if things are secure.
It would probably be good to add this as a field instead, such as `verified=true`/`verified=false`.